app/containers/RegisterEmail/index.js

```diff
@@ -244,15 +244,23 @@ class RegisterEmail extends Component {
               </View>
 
               <View style={{ flex: 1 }}>
-                <CheckBox
-                  color={'#FFF'}
-                  iconStyle={{ color: '#FFF', marginLeft: 16 }}
-                  labelStyle={{ color: '#FFF' }}
-                  label={strings.register.useReferer}
-                  size={30}
-                  checked={this.state.isChecked}
-                  onPress={this.handlePressCheckedBox}
-                />
+                <View style={{
+                    flex: 1,
```

Comment:
@denisenricohasyim93 i think there is too much whitespace, it should be better if the whitespace same with other whitespace.